Tests/TestingApps/PaywallsTester/ci_scripts/ci_pre_xcodebuild.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash -e
 
-PAYWALLS_TESTER_API_KEY=$REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY
+PAYWALLS_TESTER_API_KEY_FOR_TESTING=$REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY_FOR_TESTING
+PAYWALLS_TESTER_API_KEY_FOR_DEMOS=$REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY_FOR_DEMOS
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-if [ -z "$PAYWALLS_TESTER_API_KEY" ]; then
-    echo "PaywallsTester API key environment variable is not set."
+if [ -z "$PAYWALLS_TESTER_API_KEY_FOR_TESTING" ]; then
+    echo "PaywallsTester API key for testing environment variable is not set."
+elif
```

Comment:
Hmm this `elif` here seems wrong. Are we trying to set both keys at the same time? If so, I think we need a different condition to check both?

Reply:
lol bad copy/paste 😂 fixing